core/src/test/java/bisq/core/dao/governance/proposal/MyProposalListServiceTest.java

```diff
@@ -0,0 +1,32 @@
+package bisq.core.dao.governance.proposal;
+
+import bisq.core.btc.wallet.WalletsManager;
+import bisq.core.dao.governance.period.PeriodService;
+import bisq.core.dao.state.DaoStateService;
+
+import bisq.network.p2p.P2PService;
+
+import bisq.common.crypto.PubKeyRing;
+import bisq.common.storage.Storage;
+
+import javafx.beans.property.SimpleIntegerProperty;
+
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class MyProposalListServiceTest {
+    @Test
+    public void canInstantiate() {
+        P2PService p2PService = mock(P2PService.class);
+        when(p2PService.getNumConnectedPeers()).thenReturn(new SimpleIntegerProperty(0));
+        Storage storage = mock(Storage.class);
```

Comment:
why is storage mocked separately? It's just passed as an argument like the rest.

Reply:
yeah that test is just a stub that got me thinking about the pubkey.